.github/workflows/release.yml

```diff
@@ -70,27 +82,63 @@ jobs:
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
           SLACK_COLOR: "#BD3232"
           SLACK_ICON: https://github.com/actions.png?size=48
-          SLACK_MESSAGE: "Tests on move2kube-tests failed for move2kube on tag ${{ steps.only_tag.outputs.tag }}"
+          SLACK_MESSAGE: "Tests on move2kube-tests failed for move2kube on tag ${{ github.event.inputs.tag }}"
           SLACK_TITLE: Failed
           SLACK_USERNAME: GitHubActions
 
-  create_release_draft:
+  tag:
     needs: [run_move2kube_tests]
+    name: Tag
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.commit_ref }}
+      - id: get_sha
+        run: |
+          echo "::set-output name=sha::$(git rev-parse HEAD)"
+      - uses: actions/github-script@v3
+        with:
+          github-token: ${{ secrets.MOVE2KUBE_PATOKEN }}
+          script: |
+            github.git.createRef({
+              ...context.repo,
+              ref: "refs/tags/${{ github.event.inputs.tag }}",
+              sha: ${{ steps.get_sha.outputs.sha }}
+            })
+
+  create_release_draft:
+    needs: [tag]
     name: Create release draft
     runs-on: ubuntu-latest
     steps:
-      - id: only_tag
-        run: echo "::set-output name=tag::${GITHUB_REF#refs/tags/}"
       - uses: actions/checkout@v2
         with:
+          ref: ${{ github.event.inputs.commit_ref }}
           fetch-depth: 0
+      - id: parse_semver
+        uses: booxmedialtd/ws-action-parse-semver@v1
+        with:
+          input_string: ${{ github.event.inputs.tag }}
       - uses: actions/setup-node@v1
         with:
           node-version: "12"
       - name: create release draft
         run: |
           npm install github-release-notes@v0.17.3 -g
-          gren release --draft --prerelease --tags ${{ steps.only_tag.outputs.tag }} --username=${{ github.repository_owner }} --repo=${{ github.event.repository.name }} --token=${{ secrets.GITHUB_TOKEN }}
+          if [[ '${{ github.event.inputs.prev_tag }}' ]] ; then
+            if [[ '${{ steps.semver_parser.outputs.prerelease }}' ]] ; then
```

Comment:
Instead of copy pasting the entire command, would it be possible to have only the flag added to the string in case of pre-release, so that when we fix future bugs, we dpnt miss out on changing all commands.